crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -95,9 +78,35 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  setBlockUI('submit', $('[data-blockui]'));
-  setBlockUI('click', $('body.backups').find('[data-blockui]'));
-  setBlockUI('click', $('body.installer-upgrades').find('[data-blockui]'));
```

Comment:
yes, this needs to be adapted https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/helpers/backups_helper.rb#L42